### ISSUE TITLE
Can't generate tokens in multiplex

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/nc-multiplex#4]
With the current changes to multiplex that are in #2, it is no-longer possible to generate tokens on the management screen.

If you go to manage and attempt to generate tokens, the following appears in the text field where tokens normally appear:

```
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Error: Cannot find module &#39;undefined/app/unisys/common-session.js&#39;<br>Require stack:<br>- /Users/jdanish/gitcode/nc-multiplex/nc-multiplex.js<br> &nbsp; &nbsp;at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)<br> &nbsp; &nbsp;at Module._load (node:internal/modules/cjs/loader:920:27)<br> &nbsp; &nbsp;at Module.require (node:internal/modules/cjs/loader:1141:19)<br> &nbsp; &nbsp;at require (node:internal/modules/cjs/helpers:110:18)<br> &nbsp; &nbsp;at MakeToken (/Users/jdanish/gitcode/nc-multiplex/nc-multiplex.js:249:19)<br> &nbsp; &nbsp;at /Users/jdanish/gitcode/nc-multiplex/nc-multiplex.js:823:18<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at next (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/route.js:137:13)<br> &nbsp; &nbsp;at Route.dispatch (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/route.js:112:3)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/layer.js:95:5)</pre>
</body>
</html>
```

This error appears in the terminal:

```
...nc-multiplex: ================== Handling / MAKE TOKEN!
Error: Cannot find module 'undefined/app/unisys/common-session.js'
Require stack:
- /Users/jdanish/gitcode/nc-multiplex/nc-multiplex.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at MakeToken (/Users/jdanish/gitcode/nc-multiplex/nc-multiplex.js:249:19)
    at /Users/jdanish/gitcode/nc-multiplex/nc-multiplex.js:823:18
    at Layer.handle [as handle_request] (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/jdanish/gitcode/nc-multiplex/node_modules/express/lib/router/layer.js:95:5)


```

**Comment by daveseah** [comment]
I think this issue is closed by #2, and we can merge it into the main branch. Confirm?

**Comment by jdanish** [comment]
Confirmed!!
